app/controllers/trainees/diversity/disclosures_controller.rb

```diff
@@ -9,7 +9,7 @@ def update
         updater = Diversities::Disclosures::Update.call(trainee: trainee, attributes: disclosure_param)
 
         if updater.successful?
-          redirect_to(trainee_path(updater.disclosure.trainee))
+          redirect_to(edit_trainee_diversity_ethnic_group_path(updater.disclosure.trainee))
```

Comment:
![image](https://user-images.githubusercontent.com/470137/97008056-3f58c200-153a-11eb-83a0-c7f800442fb1.png)

Sanity check...
We going to diverge the redirects right?

Reply:
yup, that will be handled later. Just focusing on getting the whole workflow mapped together then will deal with the divergences.